localstack/services/awslambda/lambda_api.py

```diff
@@ -1060,6 +1061,50 @@ def put_concurrency(version, function):
     return jsonify(data)
 
 
+@app.route('/<version>/tags/<arn>', methods=['GET'])
+def list_tags(version, arn):
+    # the version for put_concurrency != PATH_ROOT, at the time of this
```

Comment:
I think we can remove these 2 lines of comments (same for the other functions)

Reply:
fixed